pyproject.toml

```diff
@@ -25,10 +32,10 @@ classifiers = [
 dependencies = ["pyyaml"]
 
 [project.scripts]
-dispatcher-standalone = "dispatcher.cli:standalone"
+dispatcher-standalone = "ansible_dispatcher.cli:standalone"
```

Comment:
should the executable also be renamed?

Reply:
I would like to add another `dispatcherctl` executable eventually. This does seem like a long name.

Tangential: I'm also interested to see if the entrypoints can be put in an "extras" package. So if someone doesn't install the extra (they run the service from their own script), then they don't get the entrypoint.